proto/buf/alpha/registry/v1alpha1/display.proto

```diff
@@ -41,6 +54,86 @@ service DisplayService {
   rpc ListManageableUserTemplateRoles(ListManageableUserTemplateRolesRequest) returns (ListManageableUserTemplateRolesResponse);
 }
 
+message DisplayOrganizationElementsRequest {
+  // The ID of the organization for which to check
+  // which elements should be displayed.
+  string organization_id = 1;
+}
+
+message DisplayOrganizationElementsResponse {
+  // Display create organization repository element if true.
+  bool create_repository = 1;
+  // Display create organization plugin element if true.
+  bool create_plugin = 2;
+  // Display create organization template element if true.
+  bool create_template = 3;
+  // Display organization settings element if true.
+  bool settings = 4;
+  // Display update organization settings element if true.
+  bool update_settings = 5;
+  // Display delete organization element if true.
+  bool delete = 6;
+}
+
+message DisplayRepositoryElementsRequest {
+  // The ID of the repository for which to check
+  // which elements should be displayed.
+  string repository_id = 1;
+}
+
+message DisplayRepositoryElementsResponse {
+  // Display repository settings element if true.
+  bool settings = 1;
+  // Display delete repository element if true.
+  bool delete = 2;
+}
+
+message DisplayPluginElementsRequest {
+  // The ID of the plugin for which to check
+  // which elements should be displayed.
+  string plugin_id = 1;
+}
+
+message DisplayPluginElementsResponse {
+  // Display create plugin version element if true.
+  bool create_version = 1;
```

Comment:
Is this even something you can do from the UI today? I think we do this by pushing to the registry.

Reply:
Plugins are pushed to the registry, not uploaded in a form on the frontend, but we _do_ show a slightly different UI for users who are permitted to do so, with some instructions to get started.